examples/dsl/v1/secret_sharing_sum/secret_sharing_sum_testsuite.json

```diff
@@ -0,0 +1,34 @@
+{
+    "data": [
+        {
+            "file": "examples/data/student_hetero_guest.csv",
```

Comment:
dsl/v1 & dsl/v2 & pipeline exemples should contain exactly the same examples: they should all use the same data sets and set up the same job workflow